test/k8sT/DatapathConfiguration.go

```diff
@@ -288,15 +286,8 @@ var _ = Describe("K8sDatapathConfig", func() {
 		})
 	})
 
-	Context("DirectRouting", func() {
-		BeforeEach(func() {
-			switch {
-			case helpers.IsIntegration(helpers.CIIntegrationGKE):
-			default:
-				Skip("DirectRouting without AutoDirectNodeRoutes not supported")
-			}
-		})
-
+	// DirectRouting without AutoDirectNodeRoutes not supported.
+	SkipContextIf(helpers.DoesNotRunOnGKE, "DirectRouting", func() {
```

Comment:
nit: "not supported on GKE."

Reply:
Updated to `not supported outside of GKE.`